### ISSUE TITLE
[Question] Question about random generation results for tuples

### ISSUE THREAD

**Issue by kakasoo** [samchon/typia#600]
The `typia.random` method generates tuples in the correct order when they are received, but inference at the type level is not. I don't know if this is a bug or a limitation of the type system, so I leave a question for now.

```typescript
    it('typia `random` function generate tuple', async () => {
        type ZERO_TO_SIX_TUPLE = [0, 1, 2, 3, 4, 5, 6];

        const values = typia.random<ZERO_TO_SIX_TUPLE>(); // this type is `(0 | 1 | 2 | 3 | 4 | 5 | 6)[]`

        expect(values.at(0) === 0).toBeTruthy(); // no problem
        expect(values.at(1) === 1).toBeTruthy(); // no problem
        expect(values.at(2) === 2).toBeTruthy(); // no problem
        expect(values.at(3) === 3).toBeTruthy(); // no problem
        expect(values.at(4) === 4).toBeTruthy(); // no problem
        expect(values.at(5) === 5).toBeTruthy(); // no problem
        expect(values.at(6) === 6).toBeTruthy(); // no problem
    });
```

I thought that the value of values should be [0, 1, 2, 3, 4, 5, 6], and the type inference should be the same [0, 1, 2, 3, 4, 5, 6] in the following situations. Because the union type tuple consisting of numbers from 0 to 6, and the tuple where each number and the number have a fixed index, are different.
In summary, '(0 | 1 | 2 | 3 | 4 | 5 | 6)[]' and [0, 1, 2, 3, 4, 5, 6] are different.

Thank you always for creating such a wonderful library.

**Comment by samchon** [comment]
This problem is caused by `Primitive` type.

What about fix the bug and send a PR?

https://github.com/samchon/typia/blob/master/src/Primitive.ts

**Comment by kakasoo** [comment]
OK, I'll try it today.

**Comment by kakasoo** [comment]
> [For each element of the tuple, _Primivite not applied](https://github.com/kakasoo/typia/commit/2bae392463af35ace38b8ad2979aa464cb99e987) https://github.com/samchon/typia/issues/600

If a tuple has a rapper object, `_Primitive` was not applied because it makes sense to infer it as it is.

**Comment by kakasoo** [comment]
Summary.

I decided to separate the cases of tuples from the _Primitive type and apply _Primitive to each tuple element. There was an error in the test code related to 'Atomic Class', but according to Samchon, this is a problem related to `assertParse` method, so I will send PR and close the issue.
Thank you for giving me an opportunity to contribute.

**Comment by samchon** [comment]
@kakasoo Be fixed. Upgrade to `v3.7.8` please.
